app/Http/Controllers/SeriesController.php

```diff
@@ -4,6 +4,7 @@
 
 use Illuminate\Http\Request;
 use App\Series;
+use App\Collection;
```

Comment:
Hi @gtvj this was where i had the issue. I was not able to get the collection id, which is why i was constantly getting undefined variables.